myscripts.js

```diff
@@ -0,0 +1,24 @@
+const menuBtn = document.querySelector('.openNav');
+const closeBtn = document.querySelector('.menu-list');
+const contactLink = document.querySelector('.contact-link');
+
+menuBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  document.getElementById('myLinks').style.width = '100%';
+  document.getElementById('myLinks').style.display = 'block';
+});
```

Comment:
According to the Figma design, please make sure to use the specified colors, fonts, and layouts. 😊 Use this [link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471) to follow the design template.
 
### The Figma design

<img width="312" alt="Screenshot 2023-04-23 at 13 27 49" src="https://user-images.githubusercontent.com/91237525/233856894-e6a2a39c-322e-4b92-a6e9-9fcac6a0f558.png">

<img width="309" alt="Screenshot 2023-04-23 at 13 28 49" src="https://user-images.githubusercontent.com/91237525/233856947-af1980a8-cf7c-4295-a793-f409f30a85e9.png">

### Your Design
<img width="383" alt="Screenshot 2023-04-23 at 13 28 10" src="https://user-images.githubusercontent.com/91237525/233856970-1170275f-480e-41b7-81df-e2c8594de910.png">

<img width="376" alt="Screenshot 2023-04-23 at 13 29 01" src="https://user-images.githubusercontent.com/91237525/233856980-889ddb4f-c2bd-46ec-b15a-375b991c5398.png">